### DISCUSSION TITLE
events not triggering

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#234] [Q&A] [answered]
self.add_event( "mycroft.skills.trained", self.handle_boot_finished )

the event is never received.

**Comment by mushie4282** [comment]
I've been using https://github.com/NeonGeckoCom/neon-mana-utils to see the message bus in real time. Add `self.log.info()` message to your handler method to indicate it's been triggered.
Command:
`mana tail-messagebus`

**Comment by mushie4282** [comment]
Based on https://openvoiceos.github.io/ovos-technical-manual/skills_bus/, you can run a small script to manually emit the event your code is "suppose" to recognize. I was able to trigger my skill this way.

**Comment by JarbasAl** [comment, marked as answer]
> self.add_event( "mycroft.skills.trained", self.handle_boot_finished )
> 
> the event is never received.

you probably want `mycroft.ready` event instead
